components/ImageSelector/ImageSelector.tsx

```diff
@@ -58,13 +69,13 @@ export function ImageSelector({ value, onChange }: ImageSelectorProps) {
               <div className="relative aspect-video">
                 <Image
                   src={image.src}
-                  alt={image.alt}
+                  alt={image?.alt || ""}
```

Comment:
Should not need the or here.
```suggestion
                  alt={image?.alt}
```